MultiSlider.js

```diff
@@ -389,12 +393,14 @@ export default class MultiSlider extends React.Component {
                 markerStyle={[styles.marker, this.props.markerStyle]}
                 pressedMarkerStyle={this.props.pressedMarkerStyle}
                 currentValue={this.state.valueOne}
+                valuePrefix={this.props.valuePrefix}
+                valueSuffix={this.props.valueSuffix}
               />
             </View>
           </View>
           {twoMarkers &&
           positionOne !== this.props.sliderLength &&
-          <View style={[styles.markerContainer, markerContainerTwo]}>
+          <View style={[styles.markerContainer, this.props.markerContainerStyle, markerContainerTwo]}>
```

Comment:
I think it would make sense that its after markerContainterTwo here in order to have the same behaviour on marker container 1 and two right? Otherwise you can't override the markerContainterTwo